WalletWasabi.Fluent/Views/Wallets/WalletSettingsView.axaml

```diff
@@ -35,17 +35,17 @@
         <StackPanel Spacing="10" ToolTip.Tip="Minimum anonymity score for a coin to be considered private.">
           <TextBlock Text="Privacy threshold" />
           <DockPanel>
-            <PathIcon DockPanel.Dock="Left" Foreground="{StaticResource PrivacyLevelMinimalBrush}"
-                      Data="{StaticResource privacy_minimal}" />
+            <PathIcon DockPanel.Dock="Left" Foreground="{StaticResource PrivacyLevelMediumBrush}"
```

Comment:
Should this one be:
```suggestion
            <PathIcon DockPanel.Dock="Left" Foreground="{StaticResource PrivacyLevelStrongBrush}"
```

Reply:
Yes. That is what I meant with https://github.com/zkSNACKs/WalletWasabi/pull/7843#issuecomment-1108800134